docs/_posts/gadde5300/2022-05-20-roberta_jurisbert_clas_art_convencion_americana_dh_es_3_0.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: model
+title: Spanish Text Classification (from `hackathon-pln-es`)
+author: John Snow Labs
+name: roberta_jurisbert_clas_art_convencion_americana_dh
+date: 2022-05-20
+tags: [roberta, ner, text_classification, es, open_source]
+task: Text Classification
+language: es
+edition: Spark NLP 3.4.4
+spark_version: 3.0
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+Pretrained RoBertaForSequenceClassification model, adapted from Hugging Face and curated to provide scalability and production-readiness using Spark NLP. `jurisbert-clas-art-convencion-americana-dh` is a Spanish model orginally trained by `hackathon-pln-es`.
+
+## Predicted Entities
+
+`Artículo 63.1 Reparaciones`, `Artículo 15. Derecho de Reunión`, `Artículo 4. Derecho a la Vida`, `Artículo 1. Obligación de Respetar los Derechos`, `Artículo 5. Derecho a la Integridad Personal`, `Artículo 8. Garantías Judiciales`, `Artículo 19. Derechos del Niño`, `Artículo 17. Protección a la Familia`, `Artículo 2. Deber de Adoptar Disposiciones de Derecho Interno`, `Artículo 16. Libertad de Asociación`, `Artículo 25. Protección Judicial`, `Artículo 11. Protección de la Honra y de la Dignidad`, `Artículo 12. Libertad de Conciencia y de Religión`, `Artículo 9. Principio de Legalidad y de Retroactividad`, `Artículo 7. Derecho a la Libertad Personal`, `Artículo 24. Igualdad ante la Ley`, `Artículo 6. Prohibición de la Esclavitud y Servidumbre`, `Artículo 22. Derecho de Circulación y de Residencia`, `Artículo 28. Cláusula Federal`, `Artículo 21. Derecho a la Propiedad Privada`, `Artículo_29_Normas_de_Interpretación`, `Artículo 23. Derechos Políticos`, `Artículo 13. Libertad de Pensamiento y de Expresión`, `Artículo 26. Desarrollo Progresivo`, `Artículo 30. Alcance de las Restricciones`, `Artículo 14. Derecho de Rectificación o Respuesta`, `Artículo 3. Derecho al Reconocimiento de la Personalidad Jurídica`, `Artículo 27. Suspensión de Garantías`, `Artículo 20. Derecho a la Nacionalidad`, `Artículo 18. Derecho al Nombre`
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+<button class="button button-orange" disabled>Open in Colab</button>
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/public/models/roberta_jurisbert_clas_art_convencion_americana_dh_es_3.4.4_3.0_1653049484318.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+documentAssembler = DocumentAssembler() \        .setInputCol("text") \        .setOutputCol("document")
```

Comment:
@gadde5300 Python complains about this:

![image](https://user-images.githubusercontent.com/36634572/169527817-65ae137e-ea9c-415b-b22b-68cd0f4a6ee5.png)